test/karma.conf.js

```diff
@@ -1,6 +1,8 @@
 const path = require('path');
 const webpack = require('webpack');
 
+const CI = Boolean(process.env.CI);
```

Comment:
I would rename the var to `isCI` has boolean convention `isSomething | hasItem`

Reply:
I'm not aware we have a convention for naming boolean. I believe, so far, we have used them interchangeably. Often omitting is. 

Here I think that we should aim toward importing the config file from the mono repo so I don't have to bring improvements like this manually.